apps/files_trashbin/lib/Trashbin.php

```diff
@@ -44,6 +44,7 @@
  */
 namespace OCA\Files_Trashbin;
 
+use OC\Files\Node\Node;
```

Comment:
```suggestion
use OC\Files\Node;
```
psalm seems to suggest the wrong Node is imported, but not sure if this is the right fix

Reply:
it wants the OCP type :)